tests/controls/listctrltest.cpp

```diff
@@ -151,6 +159,33 @@ void ListCtrlTestCase::ColumnClick()
 
     m_list->ClearAll();
 }
+#ifdef __WXMSW__
+void ListCtrlTestCase::SubitemRect()
+{
+    wxImageList *m_imageListNormal = new wxImageList(32, 32, true);
+    m_imageListNormal->Add( wxArtProvider::GetBitmap(wxART_ERROR) );
```

Comment:
```suggestion
    m_imageListNormal->Add(wxArtProvider::GetBitmap(wxART_ERROR));
```
..and there are other function calls later too.

Reply:
This is borderline funny because while "implementing PR suggestions" you've somehow managed to remove one of the spaces removed by the suggestion above but not the other. I really don't know how is this possible, but it is.

Of course, it's just a minor detail compared to the fact that (badly named, it shouldn't have `m_` prefix) `m_imageListNormal` is still never used, i.e. just leaked without being associated with the list control. So this test just doesn't check for anything useful at all.

I'm just going to finish this myself because it seems much simpler than continuing to list all the problems with every new submission but, frankly, I don't know how many of your PRs I will be able to review for as long as you don't review your own changes at all and don't really test them neither. Please, please try to improve the quality (not the quantity) of your contributions.

TIA!